modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/transaction/Instructions.kt

```diff
@@ -161,6 +168,73 @@ object Instructions {
         }
     }
 
+    /**
+     * Instruction for trigger to run after some asset definition for account is created
```

Comment:
I don't think we need to have this defined on a SDK core level, should be moved to some test classes code